GitExtensions.ruleset

```diff
@@ -76,7 +76,6 @@
     <Rule Id="SA1402" Action="None" />
     <Rule Id="SA1403" Action="None" />
     <Rule Id="SA1404" Action="None" />
-    <Rule Id="SA1407" Action="None" />
```

Comment:
👍 I'm a big fan of this rule. It's been known to catch bugs in old production code, so someone should review each of the changes again. Normally this applies for shift operators, but I didn't see any place in the diff where there were involved.

Reply:
The rule seems to strike a nice balance. Definitely makes things more clear.